src/deserialize/test.rs

```diff
@@ -1,47 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
We already test this in the validation tests if I'm not mistaken, so I thought it would be fine to just remove this test.

Reply:
This also asserts that desieralizing the email address lowercases it too. Can we make sure we have that assertion included somewhere else if possible?

Reply:
I'll look into it, but should be fine :)